data_utils/raw_dataset_loader.py

```diff
@@ -21,7 +21,7 @@ class RawDataLoader:
         Loads the raw files from specified path, performs the transformation to Data objects and normalization of values.
     """
 
-    def load_raw_data(self, dataset_path: str, config):
+    def load_raw_data(self, dataset_path: str, config, dataset_append=[]):
```

Comment:
```suggestion
    def load_raw_data(self, dataset_path: str, config, dataset_type: str=""):
```

Reply:
Why's not use "None"?

Reply:
Done in f5053e816c483f06ec23529e692bf0bb7419109a